### PR TITLE
APCs stay disabled until EMP timer ends

### PR DESCRIPTION
**Pull request by HMBGERDO**
## What Does This PR Do

This PR changes how APC works after being EMPd. Now APC will be unable to be turned on until the timer ends.
APC disabled time from EMP was tweaked to emp_severity * 20-40 SECONDS, which means that weak EMP will disable it for about 30 seconds, strong for 60
When APCs turns on after EMP timer ends, it will emit a silent synthetic ping sound and will say "Emergency reboot complete"

## Why It's Good For The Game

Currently, if APC is being EMPd, that just means that it is turned off. Any personnel or cyborg can enable it in couple clicks. Given that there is a timer for EMP recharge, this looks like thats not intended and it should be offline the whole time. So, this fixes it.

## Testing

Compiled, EMPd APC, tried turning it on and off.

## Changelog
:cl:
tweak: EMPd APC will be offline the whole EMP recharge time which is 20-80 seconds
/:cl:

### CODE REVIEW

File: code/modules/power/apc/apc.dm [hunk 1]
```diff
@@ -959,9 +965,21 @@
...
+	emp_disabled_time = 0
+
+	if(!operating)
+		toggle_breaker()
+
 	equipment_channel = 3
 	environment_channel = 3
+
+	playsound(loc, 'sound/machines/ping.ogg', 15, FALSE)
+	atom_say("Emergency reboot complete")
```
Comment (Henri215) [inline code/modules/power/apc/apc.dm:981]:
```suggestion
	atom_say("Emergency reboot complete.")
```

**Comment by Komrad822** [comment]
I was initially onboard with this until it was made apparent that a malf AI could be repeatedly EMP'd and thus slowly watch themselves die due to the inability to fix their APC. We dont wanna have to snowflake the AI apc. You could also emp the area and if the cameras of the area are affected, you get the same effect.

All in all a good idea but since the AI (not necessarily malf) would be super vulnerable, from a balance perspective itd be too strong of a nerf.

**Comment by matttheficus** [comment]
Ditto what Komrad said.

**Comment by S34NW** [comment]
This has three balance objections and has thus failed the vote, reasons provided are:

- **REDACTED** made a great point that itd ruin malf AIs.
- BIG EMPs on malf would be spooky. With **REDACTED** on this.
- don't think there's any way around the disastrous consequences for malf AI - there's just too many opportunities for cheese.

As such I'm closing this PR, if you think you can work around their comments feel free to reach out to the balance team and re-open this PR when ready.

**Pull request closed without merging**